packages/ui-list/src/InlineList/InlineListItem/props.ts

```diff
@@ -39,11 +39,28 @@ import type {
 } from '@instructure/shared-types'
 
 type InlineListItemOwnProps = {
-  children: React.ReactNode | ((...args: any[]) => React.ReactNode)
+  //TODO type children better e.g.:PropTypes.oneOfType([PropTypes.node, PropTypes.func])
```

Comment:
I'm not sure why this TODO is here. We only have issues with typing `Children.oneOf([ XY ])`, but `React.ReactNode | (() => React.ReactNode)` pretty much covers the original propType here.